README.md

```diff
@@ -9,7 +9,17 @@ The controller can be deployed to a cluster provided you are logged in with clus
 ```bash
 export IMG=quay.io/che-incubator/che-workspace-controller:nightly
 export TOOL=oc # Use 'export TOOL=kubectl' for kubernetes
-export WEBHOOK_ENABLED=false
+make deploy
+```
+
+By default, controller will expose workspace servers without any authentications
+which is not accessible in public available clusters.
```

Comment:
```suggestion
By default, controller will expose workspace servers without any authentication; this is not advisable for public clusters, as any user could access the created workspace via URL.
```

Reply:
fixup: Add instructions on how to deploy with webhooks and openshift oauth